cime_config/cesm/machines/config_batch.xml

```diff
@@ -424,7 +424,8 @@
 
   <batch_system MACH="yellowstone" type="lsf" version="9.1">
     <queues>
-      <queue walltimemax="24:00" jobmin="1" jobmax="8">caldera</queue>
+      <queue walltimemax="24:00" jobmin="1" jobmax="1" jobname="case.lt_archive">hpss</queue>
```

Comment:
Add a jobname attribute to the queue node.  This allows me to force the case.lt_archive script to use the hpss queue on yellowstone.